install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2377,6 +2377,15 @@ envoy:
       maxUnavailable: 2
   # -- Roll out cilium envoy pods automatically when configmap is updated.
   rollOutPods: false
+  # -- Go off-road and bring your own custom Envoy bootstrap ConfigMap. Provide the name of a ConfigMap with a `bootstrap-config.json` key.
```

```suggestion
  # -- ADVANCED OPTION: Bring your own custom Envoy bootstrap ConfigMap. Provide the name of a ConfigMap with a `bootstrap-config.json` key.
```

One small change to avoid the (very descriptive) metaphor - I've seen the use of this sort of metaphor go poorly with non-English speakers before.

---

Great catch, thank you. Just pushed changes up